ganttproject/src/main/java/net/sourceforge/ganttproject/GanttChartTabContentPanel.java

```diff
@@ -100,10 +104,14 @@ JComponent getComponent() {
   private final ContextMenu tableMenu = new ContextMenu();
   @Override
   protected Component createButtonPanel() {
-    var tableMenuButton = ToolbarKt.createButton(new TableMenuAction(), true);
+    Label filterTaskLabel  = new Label();
+    filterTaskLabel.setAlignment(Pos.CENTER_RIGHT);
+    Button tableMenuButton = ToolbarKt.createButton(new TableMenuAction(), true);
+    HBox filterComponent = new HBox(1, filterTaskLabel, tableMenuButton);
+
     Objects.requireNonNull(tableMenuButton).setOnAction(event -> {
       tableMenu.getItems().clear();
-      myTaskTableSupplier.get().tableMenuActions(new MenuBuilderFx(tableMenu));
+      myTaskTableSupplier.get().tableMenuActions(new MenuBuilderFx(tableMenu), filterTaskLabel);
```

Comment:
I suggest creating an [observable property](https://docs.oracle.com/javafx/2/binding/jfxpub-binding.htm) in the task table and listening to its changes here, instead of passing the label there.